integration-tests/velocity/src/test/java/org/apache/camel/quarkus/component/velocity/it/VelocityTest.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.velocity.it;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Map;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@QuarkusTest
+class VelocityTest {
+
+    public static final String OLD_BODY = "old_body";
+    public static final String BODY = "bar";
+    public static final String NEW_BODY = "new_body";
+    public static final String MSG = "\nDear Sheldon\n" +
+            "\n" +
+            "Thanks for the order of Camel in Action.\n" +
+            "\n" +
+            "Regards Apache Camel Riders Bookstore\n" +
+            "PS: Next beer is on me";
+
+    @Test
+    public void testTemplateViaFile() throws IOException {
+        File template = createFile("velocity_test", "Hi ${headers.name}. You have got ${headers.item}. ${body}");
+        RestAssured.given() //
```

Comment:
Couldn't we remove the trailing empty commets?

Reply:
I'll remove unnecessary comments

Reply:
done